ui/src/containers/BasicDetails/index.tsx

```diff
@@ -32,17 +32,20 @@ import Icon from '../../components/Icon';
 import TooltipDisplay from '../../components/TooltipDisplay';
 import Rating from '../Rating';
 import './BasicDetails.css';
+import { titleCase } from '../../common/titlecase';
 
 const BasicDetails: React.FC = () => {
   const { resources } = useMst();
-  const { name } = useParams();
+  const { catalog, kind, name } = useParams();
 
-  const resource: IResource = resources.resources.get(name);
+  const resource: IResource = resources.resources.get(`${catalog}/${titleCase(kind)}/${name}`);
```

Comment:
can we extract `${catalog}/${titleCase(kind)}/${name}` to a variable and reuse that variable?